streaming/java/streaming-runtime/pom.xml

```diff
@@ -1,55 +1,75 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please don't format generated pom. Or else we need top format it everytime we generated it. And indent in xml is two spaces in ray

Reply:
done.